packages/brand-positioning-block/src/settings.ts

```diff
@@ -1,18 +1,438 @@
 /* (c) Copyright Frontify Ltd., all rights reserved. */
 
-import { defineSettings } from '@frontify/guideline-blocks-settings';
+import {
+    AssetChooserObjectType,
+    AssetInputMode,
+    Color,
+    DropdownSize,
+    MultiInputLayout,
+    appendUnit,
+    defineSettings,
+    numericalOrPixelRule,
+    presetCustomValue,
+} from '@frontify/guideline-blocks-settings';
+
+import { brandItemBrandItemSizeMap, cornerRadiusMap } from './types';
+
+const WHITE_COLOR_VALUE: Color = {
```

Comment:
Can't we use fondue tokens for the colors?

Reply:
Not that I'm aware of. Not sure even if these colors exist there 😄